### PR TITLE
Add an example for using the query builder in a transaction

### PR DESCRIPTION
**Pull request by raddevon**
A user created a repo comparing a task model implementation in Knex and EdgeDB. User was under the impression it was not possible to use the querybuilder with transactions. This is part of an effort to better communicate that it is possible.

See https://discord.com/channels/841451783728529451/1038780233458909324/1039094288354652180

Companion to https://github.com/edgedb/edgedb/pull/4640

### CODE REVIEW

File: docs/querybuilder.rst [hunk 1]
```diff
@@ -174,11 +174,27 @@ Now we have everything we need to write and execute our first query!
...
+    import e from "./dbschema/edgeql-js";
+
+    const client = createClient();
+
+    const query = e.select(e.datetime_current());
+    client.transaction(async tx => {
+      const result = await query.run(tx);
+      console.log(result);
+    });
+
```
Comment (jaclarke) [inline docs/querybuilder.rst:192]: Maybe this example should be in a note block at the end of the section, so it doesn't break up the flow of the 'run a first query' mini tutorial, and make it stand out a bit. Also, probably can skip the imports and `createClient` lines.

Comment (raddevon) [reply docs/querybuilder.rst:192]: By "end of the section," do you mean at the end of the quickstart section? Would you maybe put it in the cheatsheet section as another example, or is that too buried?

Comment (jaclarke) [reply docs/querybuilder.rst:192]: Yes, end of the quickstart section.

Comment (raddevon) [reply docs/querybuilder.rst]: Thank you. Got it moved.

**Pull request merged**